File: app/src/test/java/com/example/benchmarks/models/benchmark/BenchmarkItemUnitTest.java
package com.example.benchmarks.models.benchmark;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertFalse;

import org.junit.Test;

public class BenchmarkItemUnitTest {
    @Test
    public void testUpdateBenchmarkItem() {
        BenchmarkItem item = new BenchmarkItem(1, 2, 0L, true);
        BenchmarkItem newItem = item.updateBenchmarkItem(200L);
        assertEquals(1, newItem.getNameOfCollection());
        assertEquals(2, newItem.getNameOfOperation());
        assertEquals(200L, (long) newItem.getDurationOfOperation());
        assertFalse(newItem.isProgressBarRunning());
    }
}

File: app/src/test/java/com/example/benchmarks/models/benchmark/BenchmarkItemUnitTest.kt
package com.example.benchmarks.models.benchmark

import org.junit.Assert.assertEquals
import org.junit.Assert.assertFalse
import org.junit.Test

class BenchmarkItemUnitTest {
    @Test
    fun testUpdateBenchmarkItem() {
        val item = BenchmarkItem(1, 2, 0L, true)
        val newItem = item.updateBenchmarkItem(200L)
        assertEquals(1, newItem.nameOfCollection)
        assertEquals(2, newItem.nameOfOperation)
        assertEquals(200L, newItem.durationOfOperation)
        assertFalse(newItem.isProgressBarRunning)
    }
}



